src/components/AssetScanner/WebcamScreenshot/WebcamScreenshot.tsx

```diff
@@ -20,11 +29,13 @@ export function WebcamScreenshot({
   className = '',
 }: WebcamScreenshotProps) {
   return (
-    <Wrapper
-      src={`data:image/png;base64,${src}`}
-      alt="Captured from webcam"
-      id="freezeFrame"
-      className={className}
-    />
+    <Wrapper>
+      <ImgWrapper
+        src={`data:image/png;base64,${src}`}
+        alt="Captured from webcam"
```

Comment:
This isn't internationalizable by the consuming applications.

Reply:
Better to pass string value via `strings` props (like string for reset button title)